### PR TITLE
First dev master commit

### PR DESCRIPTION
**Pull request by RabiaHaneef1**

### CODE REVIEW

File: first_Dev.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+I created the first dev file as master dev
```
Comment (ParagWankhade) [inline first_Dev.txt]: Which User story this belongs to?

Comment (RabiaHaneef1) [reply first_Dev.txt]: DATA-1157

**Review by ParagWankhade — CHANGES_REQUESTED** [review]
Change content

**Review by ParagWankhade — APPROVED** [review]
Approved

**Pull request merged**